frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/admin/ScheduleAdminPortlet.java

```diff
@@ -133,4 +142,56 @@ public void unscheduleAllServices(ActionRequest request, ActionResponse response
             log.error(e);
         }
     }
+
+    private boolean isDeleteAttachmentScheduled(ScheduleService.Iface scheduleClient, User user) throws TException {
+        RequestStatusWithBoolean requestStatus = scheduleClient.isServiceScheduled(ThriftClients.DELETE_ATTACHMENT_SERVICE, user);
+        if(RequestStatus.SUCCESS.equals(requestStatus.getRequestStatus())){
+            return requestStatus.isAnswerPositive();
+        } else {
+            throw new SW360Exception("Backend query for schedule status of deleteAttachment failed.");
+        }
+    }
+
+    @UsedAsLiferayAction
+    public void scheduleDeleteAttachment(ActionRequest request, ActionResponse response) throws PortletException, IOException {
+        try {
+            RequestSummary requestSummary =
+                    new ThriftClients().makeScheduleClient().scheduleService(ThriftClients.DELETE_ATTACHMENT_SERVICE);
+            setSessionMessage(request, requestSummary.getRequestStatus(), "Task", "schedule");
+        } catch (TException e) {
+            log.error(e);
+        }
+    }
+
+    @UsedAsLiferayAction
+    public void unscheduleDeleteAttachment(ActionRequest request, ActionResponse response) throws PortletException, IOException {
+        try {
+            User user = UserCacheHolder.getUserFromRequest(request);
+            RequestStatus requestStatus =
+                    new ThriftClients().makeScheduleClient().unscheduleService(ThriftClients.DELETE_ATTACHMENT_SERVICE, user);
+            setSessionMessage(request, requestStatus, "Task", "unschedule");
+        } catch (TException e) {
+            log.error(e);
+        }
+    }
+
+    @UsedAsLiferayAction
+    public void triggerDeleteAttachment(ActionRequest request, ActionResponse response) throws PortletException, IOException {
+        try {
+            RequestStatus requestStatus = new ThriftClients().makeAttachmentClient().deleteOldAttachmentFromFileSystem();
+            setSessionMessage(request, requestStatus, "Task", "perform");
+        } catch (TException e) {
+            log.error(e);
+        }
+    }
+
+    @UsedAsLiferayAction
+    public void triggerCveSearch(ActionRequest request, ActionResponse response) throws PortletException, IOException {
+        try {
+            RequestStatus requestStatus = new ThriftClients().makeCvesearchClient().update();
+            setSessionMessage(request, requestStatus, "Task", "perform");
+        } catch (TException e) {
+            log.error(e);
```

Comment:
It would be good to have some Error message describing the error along with Exception stacktrace.
Same for line 184, 174, 162